delta-api/src/main/java/io/cdap/delta/api/RowIdKey.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright © 2020 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.delta.api;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Represent row id keys which is uniquely determines the row in a database.
+ */
+public class RowIdKey {
+  private final List<String> rowIdKeys;
```

Comment:
Does the order of the keys matter ?

Reply:
Ordering of the keys matter from database perspective. Specifically for primary keys - https://stackoverflow.com/questions/16713233/why-does-primary-key-order-matter. Also looked through debezium code base and they are using List rather than Set for representing different keys.